microhapdb/tests/test_marker.py

```diff
@@ -70,6 +71,7 @@ def test_assumptions():
         296,  # Yu2022G2
         85,  # Yu2022G3
         64,  # Yu2022G4
+        5,  # Zhu2023
```

Comment:
These are the 5 markers that are exact duplicates of previously published markers.